README.md

```diff
@@ -61,6 +61,97 @@ make release
 ```
 
 
+## Documentation
+
+### Decoding
+
+
+These functions are intended for decoding return values from the EVM.
+
+
+* ``eth_abi.decode_single(type, data)``
+
+This function tries to decode ``data`` into the python type that corresponds
+to the provided ``type``.  This function accepts both byte strings as well as
+their hexidecimal representation with or without the ``0x`` prefix.
```

Comment:
This should be changed to state that `data` **must** be of type bytes.  Both text and hex encoded data type detection has been deprecated.